src/main/java/ru/test/restservice/service/ProjectService.java

```diff
@@ -9,25 +9,34 @@
 import ru.test.restservice.dao.UserRepository;
 import ru.test.restservice.entity.Project;
 import ru.test.restservice.entity.User;
+import ru.test.restservice.exceptions.NotFoundException;
 
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.Collections;
 import java.util.UUID;
+import java.util.stream.Collectors;
 
 
 @Service
 @RequiredArgsConstructor
 public class ProjectService {
 
+    // TODO: добавление пользователя в проект
+
     private final UserRepository userRepository;
     private final ProjectRepository projectRepository;
 
     public ModelAndView listProjects(Authentication auth) {
         ModelAndView projs = new ModelAndView("projects");
         User user = userRepository.findByEmail(auth.getName());
-        projs.addObject("projects", projectRepository.findOwnedProjects(user));
+        projs.addObject("projects",
+                projectRepository
+                        .findAll()
```

Comment:
я бы это не переносил на новую строчку, но дело вкуса
```suggestion
                projectRepository.findAll()
```